src/show.jl

```diff
@@ -23,11 +23,11 @@ function _show(pp::PlutoPlot; script_id = "pluto-plotly-div", ver = PLOTLY_VERSI
 		// Load the plotly library
 		let Plotly = undefined
 		try {
```

Comment:
It might be nice to have a try-catch in case the local `download` failed, but having it on the JS side is not enough: `get_plotly_import("$ver", "local")` would throw and the entire render would fail. So I think you need to move this try catch from JS to Julia. 

Or you can use Artifacts to be guaranteed that the import will always work 😉 and then you also don't need the "esm" and "cdn" types etc